test/bpf_conformance_runner/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ add_executable(bpftime_vm_bpf_conformance_runner
     main-bpf-conformance.cpp
 )
 
-add_dependencies(bpftime_vm_bpf_conformance_runner bpftime_vm)
-target_link_libraries(bpftime_vm_bpf_conformance_runner bpftime_vm)
+add_dependencies(bpftime_vm_bpf_conformance_runner bpftime_llvm_jit_vm)
```

Comment:
So bpf_conformance_runner can't be used for ubpf?

Reply:
I think we don't need that for bpf_conformance. Ubpf already has submodule using it.